.github/CODEOWNERS

```diff
@@ -60,6 +60,6 @@
 /plugins/dominant-color-images              @pbearne @spacedmonkey
 /tests/plugins/dominant-color-images        @pbearne @spacedmonkey
 
-# Plugin: Image Loading Optimization
-/plugins/image-loading-optimization         @westonruter
-/tests/plugins/image-loading-optimization   @westonruter
+# Plugin: Optimization Detective
+/plugins/optimization-detective         @westonruter
+/tests/plugins/optimization-detective   @westonruter
```

Comment:
Nit-pick, let's align the spacing with the other entries.

Reply:
Updated in 44a032f9738d803867a580617be92de387a765ba